--- web/src/lib/components/shared-components/version-announcement-box.svelte	2023-08-19 10:50:26.428774805 +0200
+++ web/src/lib/components/shared-components/version-announcement-box-snap.svelte	2023-08-19 10:51:08.320877048 +0200
@@ -48,20 +48,28 @@
       <p class="mb-4 text-2xl">🎉 NEW VERSION AVAILABLE 🎉</p>
 
       <div>
-        Hi friend, there is a new release of
+        You are running an community build of
         <span class="font-immich-title font-bold text-immich-primary dark:text-immich-dark-primary">IMMICH</span>,
-        please take your time to visit the
+        packaged in a
+        <span class="font-medium underline"
+          ><a href="https://snapcraft.io/immich-distribution" target="_blank" rel="noopener noreferrer"
+            >snap package</a
+          >.</span
+        >
+        The snap package should update automatically in a few days, you can inspect the progress at
+        <span class="font-medium underline"
+          ><a href="https://github.com/nsg/immich-distribution/labels/new-version" target="_blank" rel="noopener noreferrer"
+            >the issue tracker</a
+          >.</span>
+        If this breaks compability with the mobile client please provide this information, otherwise relax and give us time. Look at the
         <span class="font-medium underline"
           ><a href="https://github.com/immich-app/immich/releases/latest" target="_blank" rel="noopener noreferrer"
             >release notes</a
           ></span
         >
-        and ensure your <code>docker-compose</code>, and <code>.env</code> setup is up-to-date to prevent any misconfigurations,
-        especially if you use WatchTower or any mechanism that handles updating your application automatically.
+        for news and information of what the new release contains.
       </div>
 
-      <div class="mt-4 font-medium">Your friend, Alex</div>
-
       <div class="font-sm mt-8">
         <code>Server Version: {serverVersionName}</code>
         <br />
