diff --git a/server/src/commands/index.ts b/server/src/commands/index.ts
index 46a8d13e3..9271e4bb6 100644
--- a/server/src/commands/index.ts
+++ b/server/src/commands/index.ts
@@ -1,3 +1,4 @@
+import { CreateAdminApiKeyCommand } from 'src/commands/create-admin-api-key.command';
 import { GrantAdminCommand, PromptEmailQuestion, RevokeAdminCommand } from 'src/commands/grant-admin';
 import { ListUsersCommand } from 'src/commands/list-users.command';
 import { DisableMaintenanceModeCommand, EnableMaintenanceModeCommand } from 'src/commands/maintenance-mode';
@@ -24,4 +25,5 @@ export const commandsAndQuestions = [
   ChangeMediaLocationCommand,
   PromptMediaLocationQuestions,
   PromptConfirmMoveQuestions,
+  CreateAdminApiKeyCommand,
 ];
