diff --git a/server/src/services/database-backup.service.ts b/server/src/services/database-backup.service.ts
index f4365bb5f..5c22fe230 100644
--- a/server/src/services/database-backup.service.ts
+++ b/server/src/services/database-backup.service.ts
@@ -215,7 +215,7 @@ export class DatabaseBackupService {
     }
 
     return {
-      bin: `/usr/lib/postgresql/${databaseMajorVersion}/bin/${bin}`,
+      bin: `${process.env.SNAP}/pg-server/${bin}`,
       args,
       databaseUsername,
       databasePassword,
