diff --git a/server/src/services/database-backup.service.ts b/server/src/services/database-backup.service.ts
--- a/server/src/services/database-backup.service.ts
+++ b/server/src/services/database-backup.service.ts
@@ -205,7 +205,7 @@ export class DatabaseBackupService {
     }

     return {
-      bin: `/usr/lib/postgresql/${databaseMajorVersion}/bin/${bin}`,
+      bin: `${process.env.SNAP}/bin/${bin}`,
       args,
       databaseUsername,
       databasePassword: isUrlConnection ? new URL(databaseConfig.url).password : databaseConfig.password,
